tests/prague/eip7692_eof_v1/tracker.md

```diff
@@ -136,6 +136,18 @@
 - [ ] RJUMPV out of section bounds (ethereum/tests: ./src/EOFTestsFiller/efValidation/EOF1_rjumpv_invalid_destination_Copier.json)
 - [ ] RJUMPV into immediate (ethereum/tests: ./src/EOFTestsFiller/efValidation/EOF1_rjumpv_invalid_destination_Copier.json)
 
+### Execution
+
+- [ ] RJUMP forwards
+- [ ] RJUMP backwards (ethereum/tests: src/EIPTestsFiller/StateTests/stEOF/stEIP4200/EOF1_RJUMP_RJUMPI_RJUMPV_ExecutionFiller.yml)
+- [ ] RJUMP with 0 offset (ethereum/tests: src/EIPTestsFiller/StateTests/stEOF/stEIP4200/EOF1_RJUMP_RJUMPI_RJUMPV_ExecutionFiller.yml)
+- [ ] RJUMPI forwards with condition true/false (ethereum/tests: src/EIPTestsFiller/StateTests/stEOF/stEIP4200/EOF1_RJUMP_RJUMPI_RJUMPV_ExecutionFiller.yml)
+- [ ] RJUMPI backwards with condition true/false (ethereum/tests: src/EIPTestsFiller/StateTests/stEOF/stEIP4200/EOF1_RJUMP_RJUMPI_RJUMPV_ExecutionFiller.yml)
+- [ ] RJUMPI with 0 offset with condition true/false (ethereum/tests: src/EIPTestsFiller/StateTests/stEOF/stEIP4200/EOF1_RJUMP_RJUMPI_RJUMPV_ExecutionFiller.yml)
+- [ ] RJUMPV with different case values (ethereum/tests: src/EIPTestsFiller/StateTests/stEOF/stEIP4200/EOF1_RJUMP_RJUMPI_RJUMPV_ExecutionFiller.yml)
+- [ ] RJUMPV with case value out of table bounds (ethereum/tests: src/EIPTestsFiller/StateTests/stEOF/stEIP4200/EOF1_RJUMP_RJUMPI_RJUMPV_ExecutionFiller.yml)
+- [ ] RJUMPV with max cases number (ethereum/tests: src/EIPTestsFiller/StateTests/stEOF/stEIP4200/EOF1_RJUMP_RJUMPI_RJUMPV_ExecutionFiller.yml)
```

Comment:
I think all of these were completed in #581, in the following files:
- tests/prague/eip7692_eof_v1/eip4200_relative_jumps/test_rjump.py
- tests/prague/eip7692_eof_v1/eip4200_relative_jumps/test_rjumpi.py
- tests/prague/eip7692_eof_v1/eip4200_relative_jumps/test_rjumpv.py

Reply:
Thanks for the pointer, I marked these cases covered.